dist/index.js

```diff
@@ -3375,16 +3385,15 @@ function run() {
             // avoid having rosdep installing unrequired dependencies.
             yield execBashCommand(`diff --new-line-format="" --unchanged-line-format="" <(colcon list -p) <(colcon list --packages-up-to ${packageNameList.join(" ")} -p) | xargs rm -rf`, commandPrefix, options);
             // Install ROS dependencies for each distribution being sourced
-            for (let rosDistribution of sourceRosBinaryInstallationList) {
-                // For "latest" builds, rosdep often misses some keys, adding "|| true", to
-                // ignore those failures, as it is often non-critical.
-                yield execBashCommand(`DEBIAN_FRONTEND=noninteractive RTI_NC_LICENSE_ACCEPTED=yes rosdep install -r --from-paths src --ignore-src --rosdistro ${rosDistribution} -y || true`, commandPrefix, options);
+            if (process.platform == "linux") {
+                yield execBashCommand(`mkdir -p /opt/ros/$DISTRO`, commandPrefix, options);
```

Comment:
I was writing examplishly on prior comment - `DISTRO` is not a builtin environment variable, this will evaluate to "". Should probably be bundled above as per my other comment on this pass.